salt/utils/yamldumper.py

```diff
@@ -111,8 +95,7 @@ def dump(data, stream=None, **kwargs):
     Helper that wraps yaml.dump and ensures that we encode unicode strings
     unless explicitly told not to.
     """
-    if "allow_unicode" not in kwargs:
```

Comment:
Why are you removing this? We are essentially removing a kwarg without warning.

Reply:
looks like they are not removing the kwarg but just changed the two lines into one that does the same thing.